lib/util/cmcd_manager.js

```diff
@@ -368,6 +417,29 @@ shaka.util.CmcdManager = class {
     return (range.end - start) * 1000;
   }
 
+  /**
+   * Get the stream format
+   *
+   * @param {shaka.net.NetworkingEngine.AdvancedRequestType} type
+   *   The request's advanced type
+   * @return {(shaka.util.CmcdManager.StreamingFormat|undefined)}
+   * @private
+   */
+  getStreamFormat_(type) {
+    const AdvancedRequestType = shaka.net.NetworkingEngine.AdvancedRequestType;
+
+    switch (type) {
```

Comment:
Hm. The MSS PR adds a new advanced request type for MSS.
So whichever of these two PRs is merged first, the other will need to be updated.
Uh, unless you can't use CMCD with MSS, I'm not too knowledgeable about MSS.

Not an issue or anything, just something worth noting.

Reply:
I've added the enum value to `AdvancedRequestType` and the mapping in `CmcdManager`, that way its ready when ever the MSS PR is merged.